src/material/chips/chip-row.html

```diff
@@ -7,28 +7,25 @@
 </ng-container>
 
 
-<span class="mdc-evolution-chip__cell mdc-evolution-chip__cell--primary" role="gridcell">
-  <span
+<span class="mdc-evolution-chip__cell mdc-evolution-chip__cell--primary" role="gridcell"
     matChipAction
-    [attr.role]="editable ? 'button' : null"
     [tabIndex]="tabIndex"
```

Comment:
Isn't it problematic that focus can land on the element without it having a role?

Reply:
It has `gridcell` role, and `gridcell`s can be focused according to ARIA spec.

I admit this is not perfect, but this is the most appropriate ARIA I know of to describe the interaction for editable chips. The editable chips is a bit of a non-standard interaction pattern, but that is how it was originally designed.

Reply:
My concern was that, if I remember correctly, focus lands on the `span` which now has no role, rather than the gridcell.

Reply:
I deleted the `span` element that used to have the button role. I moved the attributes that used to be on it to the parent span, which has role gridcell.

Is it specifically browser focus? I can test the focus behavior for regressions and get back to you.

Reply:
Ah I see, I missed it while looking at the diff.